ql0.js

```diff
@@ -62,9 +62,12 @@ function parse(query) {
 function toOperator(query) {
   validate(query)
   const actualQuery = parse(query)
+  // It's important to use dedicated: false because these operators are usually
+  // created by remote peers and we don't want to give them permission to create
```

Comment:
`create` is written twice (end of L66 and start of L67). The same duplication appears in `ql1.js`.